docs/addons/upbound-universal-crossplane.md

```diff
@@ -18,37 +12,26 @@ import * as blueprints from '@aws-quickstart/eks-blueprints';
 
 const app = new cdk.App();
 
-const addOn = new blueprints.addons.UpboundUniversalCrossplaneAddOn(),
+const addOn = new blueprints.addons.UpboundUniversalCrossplaneAddOn({
+        eksMasterRole: "YOUR-MASTER-ROLE"
```

Comment:
I am afraid this will be flagged at some point. Can we call it clusterAccessRole and specify that it is actually masters in the docs?

Reply:
Yeah agree, fixed it.